Makefile

```diff
@@ -13,6 +13,10 @@ export GO15VENDOREXPERIMENT=1
 # doesn't allow +, so we use -.
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 
+DEV_ENV_IMAGE := quay.io/deis/go-dev:0.1.0
+DEV_ENV_WORK_DIR := /go/src/github.com/deis/${SHORT_NAME}
+DEV_ENV_CMD := docker run --rm -v ${PWD}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
```

Comment:
Well... for the record, the clean room thing was a bonus.  I went down this path in the first place for the sake of lowering the bar to contribution.  We've all seen (and personally experienced) the pain of getting going with go initially.  But as a secondary thing, I remain highly in favor of the benefits of a clean room.  Remember that the other half of the dependency management equation is dependency _isolation_.  Please forgive how strongly I'm about to state this opinion, but no matter what language you're working with, it's _wrong_ to allow things on to the load path (i.e. code that's on your `$GOPATH`, but not in the `vendor` dir) and to influence the build if they're not explicitly or transitivel imported by your dependency management tool-- glide in this case.  `</strong-opinion>`